subsys/net/ip/ipv6_nbr.c

```diff
@@ -1210,7 +1212,7 @@ static int handle_ns_input(struct net_icmp_ctx *ctx,
 	const struct in6_addr *na_src;
 	const struct in6_addr *na_dst;
 	struct in6_addr *tgt;
-	struct net_linkaddr_storage src_lladdr_s;
+	struct net_linkaddr src_lladdr_s;
```

Comment:
It doesn't seem we still need the second struct here? We could call `read_llao()` directly with `src_lladdr` I think?

Reply:
Yes, very true, will fix.